snap/gui/synfigstudio.desktop

```diff
@@ -0,0 +1,63 @@
+[Desktop Entry]
```

Comment:
This file is just straight from the deb but needs to be built by CMake eventually